pkg/minishift/network/dns/localdns_docker.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright (C) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dns
+
+import (
+	"fmt"
+	"github.com/docker/machine/libmachine/provision"
+
+	"github.com/minishift/minishift/cmd/minishift/cmd/config"
+	"github.com/minishift/minishift/pkg/minishift/docker"
+)
+
+const (
+	dnsmasqDefaultContainerImage = "registry.centos.org/minishift/dnsmasq"
+	dnsmasqContainerName         = "dnsmasq"
+)
+
+var (
+	dnsmasqContainerRunOptions = `--name %s \
+	--privileged \
+    -v /var/lib/minishift/dnsmasq.hosts:/etc/dnsmasq.hosts:Z \
+    -v /var/lib/minishift/dnsmasq.conf:/etc/dnsmasq.conf \
+    -v /var/lib/minishift/resolv.dnsmasq.conf:/etc/resolv.dnsmasq.conf \
+    -p '0.0.0.0:53:53/udp' \
+    -d` // {{.ResolveFilename}}, {{.AdditionalHostsPath}} --restart always
```

Comment:
Looks like some commented code is still remaining. Can we remove it?

Reply:
Leave for now. This is evolutionary code anyway, as we will work on this code later